KenticoCloud.Delivery.Tests/ContentLinkResolverTests.cs

```diff
@@ -125,7 +125,10 @@ public async void ResolveLinksInStronglyTypedModel()
             var httpClient = mockHttp.ToHttpClient();
             var resiliencePolicyProvider = new DefaultResiliencePolicyProvider(deliveryOptions);
             var contentLinkUrlResolver = new CustomContentLinkUrlResolver();
-            var contentItemsProcessor = InlineContentItemsProcessorFactory.Create(new ReplaceWithEmptyStringResolver(), new ReplaceWithEmptyStringForUnretrievedItemsResolver());
+            var contentItemsProcessor = InlineContentItemsProcessorFactory
+                .WithResolver<object, ReplaceWithEmptyStringResolver>()
+                .AndResolver<UnretrievedContentItem, ReplaceWithEmptyStringForUnretrievedItemsResolver>()
```

Comment:
Shouldn't we adjust the `ServiceCollectionExtensions` as well? And change 

https://github.com/Kentico/delivery-sdk-net/blob/master/KenticoCloud.Delivery/Extensions/ServiceCollectionExtensions.cs#L99
from
```
 services.TryAddSingleton<IInlineContentItemsResolver<object>, ReplaceWithWarningAboutRegistrationResolver>();
            services.TryAddSingleton<IInlineContentItemsResolver<UnretrievedContentItem>, ReplaceWithWarningAboutUnretrievedItemResolver>();
          
```

to 
`AddDeliveryInlineContentItemsResolver`
?

Reply:
I believe you are right, I have tested it and the way we register it now doesn't work properly, using `AddDeliveryInlineContentItemsResolver` works.

Reply:
cool, ideally, cover it with a test